working/augmentation-libraries/feature-specification.md

```diff
@@ -1,9 +1,9 @@
-# Augmentation Libraries
+# Augmentations
 
 Author: rnystrom@google.com, jakemac@google.com
 Version: 1.13 (see [Changelog](#Changelog) at end)
 
-Augmentation libraries allow splitting a Dart library into files. Unlike part
+Augmentations allow splitting a Dart library into files. Unlike part
```

Comment:
Yeah, I think I'm with Lasse on this. Part of why I called these "augmentation libraries" is to avoid the ambiguity of "augmentation" on its own.

Initially, I called them "augmentation libraries" to emphasize that (unlike part files), they are library-like in that they can have their own imports. This is less true now that they share a private scope with the main library, so removing "library" from the name probably makes sense.

Maybe "library augmentation"? So the file itself isn't a library, but it is a thing that augments a library.